packages/shared/lib/services/account.service.ts

```diff
@@ -88,12 +80,12 @@ class AccountService {
 
     /**
      * Create Account
-     * @desc create a new account and assign to the default environmenets
+     * @desc create a new account and assign to the default environments
      */
     async createAccount(name: string): Promise<DBTeam | null> {
-        const result: void | Pick<DBTeam, 'id'> = await db.knex.from<DBTeam>(`_nango_accounts`).insert({ name: name }, ['id']);
+        const result = await db.knex.from<DBTeam>(`_nango_accounts`).insert({ name }).returning('*');
 
-        if (Array.isArray(result) && result.length === 1 && result[0] != null && 'id' in result[0]) {
+        if (Array.isArray(result) && result.length === 1 && result[0]) {
```

Comment:
can the condition be simplified to `if (result?.[0])`
Same in invitations.ts

Reply:
yeah indeed, I'm not trusting 100% knex type but should work